specification.rst

```diff
@@ -2239,6 +2249,24 @@ allowed when ``cs:date`` calls a localized date format), ``cs:names``
 (delimiting names within name lists), ``cs:group`` and ``cs:layout`` (delimiting
 the output of the child elements).
 
+A delimiting element is any element as above which takes a ``delimiter`` 
+attribute, whether the attribute is supplied or not.
+
+The output of all delimiting elements is delimiter-encapsulated: no 
+``delimiter`` from a parent delimiting element will delimit between multiple 
+outputs of an inner delimiter-encapsulated element. The following produces
+``a; bc``:
+
+.. sourcecode:: xml
+
+    <group delimiter="; ">
+      <text value="a" />
+      <group>
+        <text value="b" />
+        <text value="c" />
+      </group>
+    </group>
+
```

Comment:
```suggestion
``retrieved: <http://example.com``:

.. sourcecode:: xml

    <group delimiter=": ">
      <text term="retrieved" />
      <group>
        <text term="&lt;" />
        <text variable="URL"/>
      </group>
    </group>

```

Reply:
Happy for other examples, but the specifications use realistic examples throughout and I think we should stick with that both for consistency and for readability

Reply:
Agree.